src/apis/selectThumbnailPageNumbers.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Select thumbnails in the thumbnail panel. This requires the "ThumbnailMultiselect" feature to be enabled
+ * @method WebViewer#selectThumbnailPageNumbers
```

Comment:
This should be `WebViewerInstance`. Are there any other apis that use `WebViewer`? We should change those as well.

Reply:
Thanks for catching this, I'm not sure where the `WebViewer came from